src/components/v1/scope/services/getScopeById.js

```diff
@@ -0,0 +1,13 @@
+import boom from '@hapi/boom';
+
+import scopeDao from '../DAO';
```

Comment:
change the folder's name by "dao"

Reply:
For sure! I think you already do this in your PR